README.md

```diff
@@ -62,7 +62,9 @@ expect a response code of 200
 The SSL cert can be verified
 
 #### ``` be_served_from_origin ```
-```expect(url).to be_served_from_origin origin```
+```expect(url).to be_served_from_origin(origin, custom_headers)```
```

Comment:
If In understand this correctly, `custom_headers` are requests header and not response headers, right?

If so, I find this API confusing:

`expect(url).to be_served_from_origin(origin, custom_headers)`

`custom_headers` are not part of your expectation (your expected value) and shouldn't go together with `origin`. They should be grouped with `url` (both `url` and `custom_headers` are elements of request).